messaging/WFLY-14133_configurable_bridge_call_timeout.adoc

```diff
@@ -0,0 +1,129 @@
+= Add call-timeout attribute to the Artemis core bridge resource
+:author:            Tomas Hofman
+:email:             thofman@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:          messaging,jms
+:idseparator:       -
+
+== Overview
+
+The Artemis broker defines a call-timeout attribute on the core bridge, which sets a time out for blocking calls to the
+server. Unless modified, the call-timeout defaults to 30 seconds.
+
+It was requested by GSS to make this attribute configurable via the Wildfly/JBoss EAP management model.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-14133
+
+=== Related Issues
```

Comment:
EAP7 jira could be linked here as well. I checked a few other proposals and they contains EAP7 related jira